assets/scss/settings/global/screensizes/_screensizes.scss

```diff
@@ -14,3 +14,9 @@
 $screen-large:                  1261px; // 1
 $screen-medium:                 801px;
 $screen-small:                  551px;
+
+// Custom sizes
+// -----------------------------------------------------------------------------
+$desktop-large:                 1200px;
+$desktop:                       991px;
+$tablet:                        640px;
```

Comment:
// Small devices (landscape phones, 576px and up)
@media (min-width: 576px) { ... }

// Medium devices (tablets, 768px and up)
@media (min-width: 768px) { ... }

// Large devices (desktops, 992px and up)
@media (min-width: 992px) { ... }

// Extra large devices (large desktops, 1200px and up)
@media (min-width: 1200px) { ... }

//-----
$landscape-phone: 576px;
$tablet: 768px;
$desktop: 992px;
$large-desktop: 1200px;